### PR TITLE
fix: image tag error in Helm

### PR DESCRIPTION
**Pull request by z1cheng**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

fix https://github.com/KusionStack/karpor/issues/561

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [x] Manual test (add detailed scripts or steps below)
- [ ] Other

Tested it locally:
QQ_1722355761655

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: charts/karpor/templates/_common.tpl [hunk 1]
```diff
@@ -31,6 +31,6 @@ Real image.
 */}}
 {{- define "karpor.realImage" -}}
 {{- trimPrefix "/" (list (trimAll "/" .context.Values.registryProxy)
-.repo | join "/") }}:{{ if .hasV }}v{{ end }}{{ default .context.Chart.AppVersion .tag }}
+.repo | join "/") }}:{{ if .needV }}v{{ end }}{{ default .context.Chart.AppVersion .tag }}
```
Comment (z1cheng) [inline charts/karpor/templates/_common.tpl:34]: Using the name `needV` is more appropriate for the current scenario

**Comment by elliotxx** [comment]
@z1cheng  I tested two case and they both worked fine. Good job!

image

**Comment by elliotxx** [comment]
@z1cheng Hi you still need to do a little work. 🤏 The Ci Lint is not passed because the template is modified. You need the bump chart version to `0.5.6` 👇, and then execute the `make doc` to update readme in `./charts/karpor`

image

**Comment by z1cheng** [comment]
@elliotxx Updated, please trigger the workflow again.

**Review by elliotxx — APPROVED** [review]
lgtm

**Pull request merged**